CHANGELOG.md

```diff
@@ -62,6 +63,7 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
  - Fixed `SimulateTagLowFrequencyEx` ignoring the `ledcontrol` argument (@zabszk)
  - Added ledcontrol argument to LF operations (@zabszk)
  - Added new standalone mode `dankarmulti` -  select and run multiple standalone modes (@dankar)
+ - Added `hf gallagher` commands for read/writing DESFire cards (@DarkMatterMatt)
```

Comment:
You can move this line up between current lines 9 & 10 (above " - Added support for MIFARE DESFire application creation without authentication ")